translations/translations.json

```diff
@@ -4812,7 +4812,7 @@
       "pageTitle": "ለእርስዎ አካባቢያዊ መረጃ",
       "metaDescription": "{{appName}} በጀርመን ላለው ህይወት የእርስዎ ዲጂታል መመሪያ ነው። አካባቢያዊ መረጃን፣ ክንዉኖችን እና ምክርን ያግኙ። በእርስዎ ቋንቋ ሁልጊዜ የቅርብ ጊዜ የሆነ እና የሚገኝ ነው።",
       "cityNotFound": "ያንተ ኣከባቢ ኣልተገኘምን?",
-      "clickHere": "እዚህ ተጫን"
+      "suggestToRegion": "እዚህ ተጫን"
```

Comment:
@osmers will the translators retranslate this next time and add the context if we keep it like this?
We changed the german translation from `Hier klicken` to `{{appName}} in Deiner Region vorschlagen`? Or should we better delete the whole translations?

Reply:
As far as I know we only tell them to translate the empty lines, so if we changed something and keep the old translation, then it won't be updated. Having them check all translations costs extra generally. So I would say delete the translation.

Reply:
Cool, I'll do that, thank you :)